preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,24 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+
+
+def nematic_order(gsd_file):
+
+    bond_vectors = []
+    nm_orders = []
+    nematic = freud.order.Nematic() 
```

Comment:
This also needs to be recreated for each snapshot (see comment above).

Reply:
I think nm_orders is independent of the snapshot. That is the return list of a nematic order per snapshot. I agree with the other two lines

Reply:
So for the nm_orders list the index is the frame and that is our return from the function.

Reply:
`nm_orders` should stay outside the for loops. But `nematic = freud.order.Nematic()` should be moved inside, so that it is re-created for each snapshot

Reply:
Okay, got it. Sorry, I thought your comment was highlighting both lines.

Reply:
No worries! The inline github comments can be hard to read sometimes. Technically, it's only referring to the bottom line shown in the comment